.github/workflows/prepare-release-pr.yml

```diff
@@ -11,6 +11,10 @@ on:
         description: 'Major release? (yes/no)'
         required: true
         default: 'no'
+      prerelease:
+        description: 'Prerelease (ex: "rc1", including the quotes)'
+        required: true
+        default: '""'
```

Comment:
the quoting on this seems weird -- I would expect `default: ''` to be better (coupled with a suggested change below)

Reply:
Done, thanks!